src/test/java/com/programmers/application/domain/customer/CustomerTest.java

```diff
@@ -110,18 +74,72 @@ void changeName(String changeName) {
     @DisplayName("변경 할 이름이 비어있거나 없으면, changeName()을 실행할 때 예외가 발생한다.")
     @ParameterizedTest
     @NullAndEmptySource
-    void throwNullAndEmptyNameException(String changeName) {
+    void throwNullAndEmptyChangeNameException(String changeName) {
```

Comment:
```suggestion
    void throwExceptionWhenChangeNameToNullOrEmpty(String changeName) {
```
이런식으로 이름을 지어주는 것이 좋아보여요. 결과, 행위, 가정이 한 눈에 들어오지 않고 문장이 조금 어색해서요.
throw다음에는 어떤 예외가 나와야 할 거 같은데, 가정과 행위가 나오네요.